stl/inc/xmemory

```diff
@@ -1446,22 +1515,23 @@ class _NODISCARD _Uninitialized_backout_al {
     using pointer = _Alloc_ptr_t<_Alloc>;
 
 public:
-    _Uninitialized_backout_al(pointer _Dest, _Alloc& _Al_) : _First(_Dest), _Last(_Dest), _Al(_Al_) {}
+    _CONSTEXPR20_DYNALLOC _Uninitialized_backout_al(pointer _Dest, _Alloc& _Al_)
+        : _First(_Dest), _Last(_Dest), _Al(_Al_) {}
 
     _Uninitialized_backout_al(const _Uninitialized_backout_al&) = delete;
     _Uninitialized_backout_al& operator=(const _Uninitialized_backout_al&) = delete;
 
-    ~_Uninitialized_backout_al() {
+    _CONSTEXPR20_DYNALLOC ~_Uninitialized_backout_al() {
         _Destroy_range(_First, _Last, _Al);
     }
 
     template <class... _Types>
-    void _Emplace_back(_Types&&... _Vals) { // construct a new element at *_Last and increment
+    _CONSTEXPR20_DYNALLOC void _Emplace_back(_Types&&... _Vals) { // construct a new element at *_Last and increment
         allocator_traits<_Alloc>::construct(_Al, _Unfancy(_Last), _STD forward<_Types>(_Vals)...);
         ++_Last;
     }
 
-    pointer _Release() { // suppress any exception handling backout and return _Last
+    _CONSTEXPR20_DYNALLOC pointer _Release() { // suppress any exception handling backout and return _Last

```

Comment:
```suggestion
    constexpr pointer _Release() { // suppress any exception handling backout and return _Last
```
In similar `_Release()` functions, you make them simply `constexpr` rather than `_CONSTEXPR20_DYNALLOC`, presumably because it does not directly require any functionality provided by WG21-P0784.